SoundUtils1/Class1.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SoundUtils1
+{
+    public class Class1
+    {
+        public void AttacksBeep()
+        {
+            Console.Beep(400, 100);
+            Console.Beep(500, 100);
+            Console.Beep(600, 100);
+            Console.Beep(400, 100);
+        }
+
+        public void HealBeeps()
+        {
+            Console.Beep(800, 100);
+            Console.Beep(700, 100);
+            Console.Beep(600, 100);
+            Console.Beep(700, 100);
+        }
+
+        public void AcientPianoBeeps()
```

Comment:
Nit: Ancient

Reply:
it is acient

Reply:
### Acient

Reply:
### 

> _**Really? It is??**_

Reply:
This is a test
`Console.WriteLine("Test");`